idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -298,6 +298,21 @@ service WorkflowService {
       4: shared.ServiceBusyError serviceBusyError,
     )
 
+  /**
+  * SignalWithStartWorkflowExecution is used to ensure sending a signal event to a workflow execution.
+  * If workflow is running, this results in WorkflowExecutionSignaled event recorded in the history
```

Comment:
this results in WorkflowExecutionSignaled event recorded in the history -> this results in a WorkflowExecutionSignaled event being recorded in the history